packages/webview_flutter/webview_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.2.3
+
+* Updates code sample.
```

Comment:
nit: This could be more descriptive 

```suggestion
* Fixes the code sample in the dartdocs for `WebViewController.addJavaScriptChannel`. 
```

Reply:
`version_check` Test Failed with `When bumping the version for release, the NEXT section should be incorporated into the new version's release notes.`